src/printer/index.js

```diff
@@ -965,7 +965,17 @@ function genericPrint(path, options, print) {
       // on python2 exc is called type
       const type = n.type ? "type" : "exc";
 
-      return group(concat(["raise", line, path.call(print, type)]));
+      return concat([
+        "raise",
+        group(
+          concat([
```

Comment:
nit: I've been using the `groupConcat` and `indentConcat` functions that I made, since those are very common things to want to do together, and I think it makes it a bit more readable (not that this is unreadable though 🙂)